docs/develop/example_subjects_extension.md

```diff
@@ -1,148 +0,0 @@
-# Example Subjects Extension
```

Comment:
I feel a bit bad about removing this... but it does not match the vocabularies section nor much the extensions one as it is now... However, I think we should keep it somehow somewhere... open to ideas :)

Reply:
When we bring the extensions section back, I will bring it back.